plugins/gssapi.c

```diff
@@ -399,6 +399,7 @@ sasl_gss_encode(void *context, const struct iovec *invec, unsigned numiov,
     }
     
     if (output_token->value && output) {
+	int len;
```

Comment:
I think this be uint32_t, int is really icky here. I think int can still be 2 bytes on some platforms.